docs/upgrade.html

```diff
@@ -60,6 +60,7 @@
     <li> The new Java consumer is no longer in beta and we recommend it for all new development. The old Scala consumers are still supported, but they will be deprecated in the next release
          and will be removed in a future major release. </li>
     <li> The BrokerState "RunningAsController" (value 4) has been removed. Due to a bug, a broker would only be in this state briefly before transitioning out of it and hence the impact of the removal should be minimal. The recommended way to detect if a given broker is the controller is via the kafka.controller:type=KafkaController,name=ActiveControllerCount metric. </li>
+    <li> Cluster Id has been added. It will be generated automatically on upgrading and restarting the brokers. It is available as kafka.server:type=KafkaServer,name=ClusterId metric and is also part of the Metadata response. Any class can get access to cluster id by implementing the ClusterResourceListener interface. </li>
     <li> The new Java Consumer now allows users to search offsets by timestamp on partitions.
```

Comment:
Can you please switch this line with the `BrokerState` line? (I know it's unrelated to this PR)